src/archives.py

```diff
@@ -64,46 +63,50 @@ def extract_archive_task(
     """
     input_files = get_input_files(pipe_result, input_files or [])
     output_files = []
+    log_files = []
     file_filters = task_config.get("file_filter") or []
     if file_filters:
         file_filters = file_filters.split(",")
 
+    # Send a task progress event to indicate the task has started
+    self.send_event("task-progress")
+
     for input_file in input_files:
         log_file = create_output_file(
             output_path,
-            display_name=f"extract_archives_{input_file.get("display_name")}.log",
+            display_name=f"extract_archives_{input_file.get('display_name')}.log",
         )
 
         (command_string, export_directory) = extract_archive(
             input_file, output_path, log_file.path, file_filters
         )
 
         if os.path.isfile(log_file.path):
-            output_files.append(log_file.to_dict())
+            log_files.append(log_file.to_dict())
 
         export_directory_path = Path(export_directory)
-        extracted_files = [f for f in export_directory_path.glob("**/*") if f.is_file()]
+        extracted_files = [file for file in export_directory_path.glob("**/*") if file.is_file()]
         for file in extracted_files:
             original_path = str(file.relative_to(export_directory_path))
-
             output_file = create_output_file(
                 output_path,
                 display_name=file.name,
                 original_path=original_path,
-                data_type=f"worker:openrelik:extraction:archive",
+                data_type="extraction:archive:file",
```

Comment:
* do you want to add the work ("extraction") here or do you want to do that in the mediator for all received output_file results from every worker process?

* These are extracted files so I don't know how useful the "archive" date_type element here is. On the other hand it can not hurt to include it as consumers later in the workflow can set their filter to e.g. "*:file:*" and get all files (independent if they were extracted or coming from another worker).

Reply:
Ack - yeah I will change this as soon as we have the new data_type enums merged and ready. I will update all our workers then.

OK to keep as this for now?